modules/service.sh

```diff
@@ -21,6 +21,7 @@ service_enable() {
 
     check_user
     service_check_support "$service"
+    _service_check_enabled "$service"
```

Comment:
I think we should `|| exit 6` here and change the manpage accordingly. Currently 6 is only for "fips is already enabled", it can now be "$service is already enabled". Or, if we don't want to change existing meaning, we could create a new exit status.

Reply:
yeah that makes sense, I'll change it to `exit 6`

Reply:
Should we perhaps move the enabled check to after the token check? Just noticed this:
```
ubuntu@ubuntu:~$ sudo ua enable-livepatch 
Canonical Livepatch is already enabled
ubuntu@ubuntu:~$ 
```

Reply:
Maybe, the only downside is that in the case of ESM the token is checked with an HTTP get, which takes a bit more time. I think either way is fine, maybe file a separate bug?

Reply:
Ah, right, I thought it was just a simple presence check, or syntax. Didn't realize the validity of the token was included. Leave it like this then.